### PR TITLE
Add Zerolog integration

### LINKED ISSUE #853: Add support for zerolog
## Summary

Support for [zerolog](https://github.com/rs/zerolog) would be great. It has similar Hook approach to logrus which is already supported.

## Motivation

It's a widely used logging package and would help a lot with tools that are using it.

### PR DESCRIPTION
**Pull request by ribice**
Resolves #853

### CODE REVIEW

**Comment by ribice** [comment]
Instead of implementing the zerolog.Writer, alternative would be to use Hooks (similar to logrus implementation).

**Comment by ribice** [comment]
For the ./otel `go mod tidy` issue, I locally cd into otel and do `go mod tidy` - nothing happens.

**Comment by antisocrates** [comment]
Will this be available in the next minor release?

**Comment by ribice** [comment]
> Will this be available in the next minor release?

Yes, we plan to include this in next release.

**Review by cleptric — DISMISSED** [review]
Let's make this it's own package. You can update https://github.com/getsentry/sentry-go/blob/master/.craft.yml and I'll add it to our release registry.

**Review by cleptric — COMMENTED** [review]
Meant to comment

File: Makefile [hunk 1]
```diff
@@ -79,4 +79,4 @@ lint: ## Lint (using "golangci-lint")
 
 fmt: ## Format all Go files
 	gofmt -l -w -s .
-.PHONY: fmt
+.PHONY: fmt
```
Comment (cleptric) [inline Makefile:82]:
```suggestion
.PHONY: fmt

```

**Review by cleptric — APPROVED** [review]

**Pull request merged**